docs/UserGuide.md

```diff
@@ -1,172 +1,191 @@
+    - 5.4 [Commands for mappings](#54-commands-for-mappings)
+        - 5.4.1 [List all mappings: `mapping list`](#541-list-all-mappings-mapping-list)
+        - 5.4.2 [Add a mapping: `mapping add`](#542-add-a-mapping-mapping-add)
+        - 5.4.3 [Delete a mapping: `mapping delete`](#543-delete-a-mapping-mapping-delete)
+        - 5.4.4 [Search for a mapping by an attribute: `mapping search`](#544-search-a-mapping-by-attributes-mapping-search)
+        - 5.4.5 [Sort all mappings by an attribute: `mapping sort`](#545-sort-a-mapping-by-attributes-mapping-sort)
+    - 5.5 [Commands for notes](#55-commands-for-notes)
+        - 5.5.1 [List all notes: `note list`](#551-list-all-notes-note-list)
+        - 5.5.2 [Add a note: `note add`](#552-add-a-note-note-add)
+        - 5.5.3 [Delete a note: `note delete`](#553-delete-a-note-note-delete)
+        - 5.5.4 [Update a note: `note update`](#554-update-a-note-note-update)
+        - 5.5.5 [Tag a note: `note tag`](#555-tag-a-note-note-tag)
+        - 5.5.6 [Clear all tags from a note: `note cleartag`](#556-clear-all-tags-in-a-note-note-cleartag)
+    - 5.6 [View help : `help`](#56-view-help--help)
+    - 5.7 [Exit SEPlendid: `exit`](#57-exit-seplendid--exit)
+    - 5.8 [Save the data](#58-save-the-data)
 6. [FAQ](#6-faq)
 7. [Command summary](#7-command-summary)
 
 </div>
-
+<br>
+<br>
 --------------------------------------------------------------------------------------------------------------------
 
 ## 1. How to navigate this User Guide?
 
-As a new user in SEPlendid, this user guide serves as an easy-to-follow guide in aiding you in executing your first 
-commands in SEPlendid, utilising SEPlendid to it's fullest potential!
+As a new user in SEPlendid, this user guide serves as an easy-to-follow guide, guiding you through the execution of your
+first commands in SEPlendid. You can unlock SEPlendid full potential in this guide!
+
+Here is a step-by-step instruction for you to navigate the user guide:
 
-Here is a step-by-step instruction in navigating this user guide for **new** users:
-1. Download SEPlendid by heading over to the [installation](#) section which provides a guided tour for you to get
-   started with SEPlendid.
-2. Get familiar with SEPlendid by heading over to the [Quick Reference Guide](#) section which shows you how to use 
-navigate SEPlendid efficiently.
+1. Download SEPlendid in your device by heading over to the [Installation](#3-installation) section.
+2. Quickly familarise yourself with SEPlendid by heading over to the [Quick Reference Guide](#4-quick-reference-guide)
+   section.
 
-If you are an **experienced** user, you can head over to the [Command Summary](#) section for a well-curated overview
-of the commands available in SEPlendid.
+If you are an **experienced** user, you can head over to the [Command summary](#7-command-summary) section for a
+well-curated overview of the commands available in SEPlendid.
 
-<br />
-<br />
+<br>
+<br>
```

Comment:
Same here.

Reply:
I wouldn't mind reverting this changes. But the links in this changes does not work. Would you like me to revert it back to you non-functioning version for you to create a PR later?

Reply:
Although I do agree some of the changes in the wording is not necessary

Reply:
No need to revert, my concern is on authorship. Just want to make sure everyone gets credit for what they wrote.